src/features/auth/ui/RegionModal.tsx

```diff
@@ -288,11 +292,21 @@ const RegionModalCloseButton = ({
 };
 
 export const RegionModal = ({ onClose }: { onClose: () => Promise<void> }) => {
+  useEffect(() => {
+    // 모달이 닫힌 후엔 모달 내부 내용을 초기화 하는 클린업 함수
+    return () => {
+      useAddressModalStore.setState({
+        keyword: "",
+        position: { lat: 0, lng: 0 },
+        origin: "keyword",
+      });
+    };
+  }, []);
```

Comment:
어디서 본 꿀팁인데 store를 초기화하는 action을 만들더라구요!

```javascript
const initialAddressModalStore = {
  keyword: "",
  position: { lat: 0, lng: 0 },
  origin: "keyword",
}
export const useStore = create(set => ({
  ...initialAddressModalStore,
  
  setKeyword: (keyword) => set({ keyword }),
  setPosition: (position) => set({ position }),
  setOrigin: (origin) => set({ origin }),
  resetStore: () => set(initialAddressModalStore)
}))
```

useEffect 내부에서 사용하게 되면
```suggestion
  useEffect(() => {
    return () => {
      resetStore();
    };
  }, [resetStore]);
```

Reply:
오 좋네요 적용하겠습니다